EIPS/eip-6239.md

```diff
@@ -0,0 +1,167 @@
+---
+eip: 6239
+title: Semantic Soulbound Tokens
+description: Adding RDF triples to EIP-721 and EIP-5192 metadata to capture the social meaning of the Token
+author: Jessica Chang (@JessicaChg)
+discussions-to: https://ethereum-magicians.org/t/eip-6239-semantic-soulbound-tokens/12334
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-12-30
+requires: 165, 721, 5192
+---
+
+## Abstract
+
+This proposal extends [EIP-721](./eip-721.md) and [EIP-5192](./eip-5192.md) by introducing a standard for adding RDF triples to Soulbound Tokens (‘SBTs‘) metadata.
+
+Soulbound Token represents the commitments, credentials, and affiliations of accounts. Resource Description Framework (‘RDF’) is a standard data model developed by the World Wide Web Consortium (‘W3C’) and is used to represent information in a structured, machine-readable format in triples consisting of a subject, a predicate, and an object (the ‘RDF triple’). These triples can be combined and linked together to represent more complex information and relationships. Semantic SBTs are built on existing [EIP-721](./eip-721.md) and [EIP-5192](./eip-5192.md) standards to include RDF triples in metadata to capture and store the meaning of social metadata as a network of accounts and attributes.
+
+Semantic SBT provides a foundation for publishing, linking, and integrating data from multiple sources, and enables the ability to query and retrieve information across these sources, using inference to uncover new insights from existing social relations. For example, form the on-chain united social graph, assign trusted contacts for social recovery, and supports fair governance.
+
+## Motivation
+
+While the existence of SBTs can create a decentralized social framework, there still needs to be a way to create the connectedness of SBTs through relationships on-chain. Current NFTs and SBTs are hosting data off-chain, some even on centralized servers, let alone the linking of social data.  And to further fuel the boom of the SBTs ecosystem, we need a bottom-up and decentralized way to maintain people’s social identity related information.
```

Comment:
> Current NFTs and SBTs are hosting data off-chain, some even on centralized servers, let alone the linking of social data.

I don't really understand what this sentence is trying to say. Would you mind rewording it?